src/client/components/editor/index.js

```diff
@@ -329,16 +342,26 @@ class Editor extends React.Component {
     }
   }
 
+  resetMenuState(){
+    this.setData({
+      drawMode: false,
```

Comment:
This won't work (no events)

You need toggleDrawMode(false)

Reply:
Okay I changed that, but I also notice that the tooltips are activated before the close task list animation ends, leading to the button tooltips shifted and floating in the middle of the editor.  Thoughts on how I might solve this?  I was thinking I could try to use setInterval.

Reply:
I'm not noticing that when I pull in your changes and test locally.  What are the steps to reproduce the issue?